clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java

```diff
@@ -84,41 +84,23 @@ void loggingWithTraceIdReturnsOk() throws Exception {
   }
 
   @Test
-  void loggingWithTimestampSet() throws Exception {
-    mockMvc.perform(GET_REQUEST_BUILDER_WITH_EMAIL_AND_TRACE_ID_AND_JVS_TOKEN);
-    verify(loggingClient).log(auditLogRequestCaptor.capture());
-    assertThat(auditLogRequestCaptor.getValue().getTimestamp()).isEqualTo(TEST_TIMESTAMP);
+  void loggingWithoutInterceptedEmailCausesBadRequestError() throws Exception {
+    mockMvc.perform(GET_REQUEST_BUILDER).andExpect(status().isBadRequest());
   }
 
   @Test
-  void loggingUsesTheEmailInRequestAuthenticationInfo() throws Exception {
+  void loggingWithCustomInfoAndAssertResult() throws Exception {
```

Comment:
Can we merge `loggingWithTraceIdReturnsOk() throws Exception` in this test case?